packages/app-store/routing-forms/components/_components/Header.tsx

```diff
@@ -141,6 +141,18 @@ const Actions = ({
               StartIcon="download">
               {t("download_responses")}
             </FormAction>
+            {form?.id && form?.teamId && (
```

Comment:
aside: we need to decide whether or not to include `form?.teamId` condition here. discussing in slack.